code/go/0chain.net/chaincore/chain/config.go

```diff
@@ -38,6 +40,18 @@ const (
 )
 
 type Config interface {
+	State() bool
+	Dkg() bool
+	ViewChange() bool
+	BlockRewards() bool
+	Storage() bool
+	Faucet() bool
+	Interest() bool
+	// Indicates is fees enabled
+	Miner() bool
+	Multisig() bool
+	Vesting() bool
+	Zcn() bool
```

Comment:
What about their corresponding values in 0chain.yaml and viper. Do they need to be changed or I leave it as before:

https://github.com/0chain/0chain/blob/1b24b8d460cd4d281ad20fc357fee7b6b5d11ba2/docker.local/config/0chain.yaml#L101-L107

https://github.com/0chain/0chain/blob/89884cb0fdf825d04df1c1d6967b633512c8baf4/code/go/0chain.net/chaincore/chain/config.go#L514-L524

I can do them like this:
https://github.com/0chain/0chain/blob/1b24b8d460cd4d281ad20fc357fee7b6b5d11ba2/docker.local/config/0chain.yaml#L89-L93

Reply:
Leave the yaml file unchanged, otherwise we will need to update deployment scripts to adjust the changes, should only need to change the function names.